src/components/Sidebar/SidebarItem/SidebarItem.tsx

```diff
@@ -0,0 +1,457 @@
+        active: theme.palette.monoA.A150
+      }),
+
+      '&.Mui-selected': {
+        ...theme.mixins.listItem({
+          background: theme.palette.monoA.A100,
+          hover: theme.palette.monoA.A50,
+          icon: theme.palette.monoA.A600,
+          focus: theme.palette.monoA.A75,
+          active: theme.palette.monoA.A150
+        })
+      }
+    }),
+
+    ...(ownerState.color === 'primary' && {
+      ...theme.mixins.listItem({
+        background: 'transparent',
+        color: theme.palette.monoB.A800,
+        icon: theme.palette.monoB.A800,
+        hover: theme.palette.monoB.A50,
+        focus: theme.palette.monoB.A200,
+        active: theme.palette.monoB.A150
+      }),
+
+      '&.Mui-selected': {
+        ...theme.mixins.listItem({
+          background: theme.palette.monoB.A100,
+          color: theme.palette.monoB[500],
+          icon: theme.palette.monoB[500],
+          hover: theme.palette.monoB.A50,
+          focus: theme.palette.monoB.A75,
+          active: theme.palette.monoB.A150
+        })
+      }
+    })
+  }
+}));
+
+const SidebarItemSecondaryAction = styled(IconButton, {
+  name: 'ESSidebarItem',
+  slot: 'SecondaryAction',
+  overridesResolver: (props, styles) => styles.secondaryAction
+})<{ ownerState: SidebarItemOwnerState }>(({ theme, ownerState }) => ({
+  '&.MuiIconButton-root': {
+    position: 'absolute',
+    left: `${ownerState.width}px`,
+    top: '8px',
+    zIndex: '5',
+
+    ...(ownerState.color === 'default' && {
+      ...theme.mixins.button({
+        background: 'transparent',
+        color: theme.palette.monoA.A500,
+        hover: theme.palette.monoA.A50,
+        focus: theme.palette.monoA.A200,
+        active: theme.palette.monoA.A150
+      })
+    }),
+
+    ...(ownerState.color === 'primary' && {
+      ...theme.mixins.button({
+        background: 'transparent',
+        color: theme.palette.monoB.A800,
+        hover: theme.palette.monoB.A50,
+        focus: theme.palette.monoB.A200,
+        active: theme.palette.monoB.A150
+      })
+    }),
+
+    '& .MuiSvgIcon-root': {
+      transform: `${ownerState.isNestedMenuOpen ? 'rotate(270deg)' : 'none'}`
+    }
+  }
+}));
+
+const SidebarItemTooltip = styled(
+  ({ className, ...props }: TooltipProps) => <Tooltip {...props} classes={{ popper: className }} />,
+  {
+    name: 'ESidebarItem',
+    slot: 'Tooltip',
+    overridesResolver: (props, styles) => styles.tooltip
+  }
+)(({ theme }) => ({
+  '&[data-popper-placement*="right"] .MuiTooltip-tooltip': {
+    padding: '0',
+    maxWidth: '288px',
+    background: theme.palette.monoA.A600,
+    borderRadius: '4px',
+
+    '& .MuiTooltip-arrow': {
+      color: theme.palette.monoA.A600,
+
+      '&::before': {
+        borderBottomLeftRadius: '2px'
+      }
+    }
+  },
+  '&[data-popper-reference-hidden]': {
+    pointerEvents: 'none',
+    opacity: '0'
+  }
+}));
+
+const SidebarItemTooltipDivider = styled(Divider, {
+  name: 'ESSidebarItem',
+  slot: 'TooltipItem',
```

Comment:
```suggestion
  slot: 'TooltipDivider',
```
Видимо не там от айтема вписал.